### DISCUSSION TITLE
Different Pricing Same License Number

### DISCUSSION THREAD

**Discussion by rhodema** [CMSgov/hospital-price-transparency#121] [Q&A] [answered]
Our organization has outpatient locations under the same license number but with different pricing. Are we able to list the different prices in columns in our main file? Or do they need to be separate files? If we can list as columns in the main file, where in the format is it appropriate to order them?

**Comment by RidenourK** [comment, marked as answer]
@rhodema. Thank you for your question. 

Under § 180.50(a)(2), each hospital location operating under a single hospital license (or approval) that has a different set of standard charges than the other location(s) operating under the same hospital license (or approval) must separately make public the standard charges applicable to that location. Under § 180.50(d), hospitals must ensure that the standard charge information in the MRF is “clearly identified with the hospital location with which the standard charge information is associated.” As we explained in the CY 2020 HPT final rule, we believed it would be sufficient for a hospital to post a single file of standard charges for a single campus location, if the file includes charges for all items and services offered at the single campus location. **In cases where such off-campus and affiliated sites operate under the same license (or approval) as a main location but have different standard charges or offer different items and services, these locations would separately make public the standard charges for such locations** ([84 FR 65564](https://www.federalregister.gov/citation/84-FR-65564)).
